### DISCUSSION TITLE
Nedis Zigbee Remote Control ZBRC10WT

### DISCUSSION THREAD

**Discussion by jnswb** [Koenkk/zigbee2mqtt#22600] [Devices]
I have Nedis Remote that is identified as 

https://www.zigbee2mqtt.io/devices/TS0215A_remote.html

It works fine and I'm using it, but as the button assignments on my remote are more generic 0, 1, A and B (see image) the action names disarm, arm_day_zones, arm_night_zones, arm_all_zones kind of make no sense.

Is there a solution to this?

<img width="195" alt="Nedis-remote-control" src="https://github.com/Koenkk/zigbee2mqtt/assets/13587638/e592ba21-8ff9-448b-a14d-e3bd9887c647">

**Comment by Gunni** [comment]
I have the same problem, I want to **bind** it to stuff like lights...

**Comment by andrei-lazarov** [comment]
The action names in Z2M can be anything. 
But right now it's too late to change them. It will break people's automations.

They were chosen to reflect the commands the remote sends.
It does in fact send `commandArm` from cluster `ssIasAce` etc.

@Gunni You can not bind it to lights, because it does not send On, Off, Move commands from the `genOnOff` and `genLevelCtrl` clusters.
Binding happens outside Z2M. So renaming the action in Z2M has no effect in this case.
Here is an example that works: [IKEA RODRET](https://www.zigbee2mqtt.io/devices/E2201.html)

**Reply by Gunni** [reply]
I kinda onew that, guesss I have to hope for a firmware hack?

**Reply by andrei-lazarov** [reply]
Yes! Firmware hacking is the way to go 🙂 

We are trying to make it unified for multiple device types in this repo. Remotes coming soon:
- [romasku/tuya-zigbee-switch](https://github.com/romasku/tuya-zigbee-switch)

Otherwise, these guys do it per device:
- [slacky1965/tuya_battery_switch_ts004x_zed](https://github.com/slacky1965/tuya_battery_switch_ts004x_zed)
- [pvvx](https://github.com/pvvx)

**Reply by Gunni** [reply]
I wonder if it can handle esphome with thread.

**Reply by andrei-lazarov** [reply]
Find out the chip it uses. Use MAC lookup and/or open it up.
If it has Silabs EFR32MG21 ([ZS3L](https://developer.tuya.com/en/docs/iot/zs3l?id=K97r37j19f496)), I'd say it's capable of Thread.

Otherwise probably not with the older Telink TLSR825x, and Silabs MG1
